src/Winp/Form/ServiceForm.cs

```diff
@@ -84,7 +85,14 @@ public ServiceForm()
         _packagePhpStatusLabel.ImageList = statusImageList;
         _packagePhpMyAdminStatusLabel.ImageList = statusImageList;
         _services = [mariaDbService, nginxService, phpService, phpMyAdminService];
-        _scheduler = scheduler;
+
+        // Configure NotifyIcon (it's initialized in Designer.cs)
+        if (_notifyIcon != null)

```

Comment:
I don't get how `_notifyIcon` can be `null` here: since `InitializeComponent` was called we're guaranteed it's defined, isn't it?